### ISSUE TITLE
Trouble shooting Start task failed 

### ISSUE THREAD

**Issue by ctlamb** [Azure/doAzureParallel#327]
For some reason some of my VMs fail to start their task. Seems weird seeing as they should all be the same and receive the same info. How should I go about trouble shooting this? I see there is no stderr or anything for these nodes that I can look at.

Thanks!

![screen shot 2018-11-15 at 1 03 48 pm](https://user-images.githubusercontent.com/34384805/48635121-84564000-e984-11e8-937e-ab082a87cad7.png)

```
clusterConfig <- list(
  "name" = "LambRaster",
  "vmSize" = "Standard_E8_v3",
  "maxTasksPerNode" = 1,
  "poolSize" = list(
    "dedicatedNodes" = list(
      "min" = 1,
      "max" = 125
    ),
    "lowPriorityNodes" = list(
      "min" = 0,
      "max" = 0
    ),
    "autoscaleFormula" = "QUEUE"
  ),
  "containerImage" = "rocker/geospatial:latest",
  "rPackages" = list(
    "cran" = c("doParallel", "here", "dismo", "gbm", "snow"),
    "github" = c("Azure/doAzureParallel"),
    "bioconductor" = c()
  ),
  "commandLine" = list()
)



rast.results <- foreach(i = 1:nrow(bp),.packages = c("doParallel", "here", "dismo", "gbm", "snow", "doAzureParallel"), .errorhandling="pass",
                        .options.azure = list(enableCloudCombine=FALSE,
                                              job = job_name)) %dopar% {
```

**Comment by ctlamb** [comment]
Fantastic, thanks for running this, @brnleehng, finding the issue, and suggesting solutions and Batch Explorer. Really appreciate it. Very helpful.

I created a Docker file, with the help of your instructions ([here](https://github.com/Azure/doAzureParallel/blob/master/docs/33-building-containers.md)). It took me a bit to figure it out, but I built my docker off the [geospatial rocker](https://github.com/rocker-org/geospatial/blob/master/3.5.0/Dockerfile). I got it all working. Everything seems to be working now.

Thanks!
